lib/v2/screens/profile_screens/guardians/components/my_guardians_tab.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/v2/datasource/remote/model/firebase_models/guardian_model.dart';
+import 'package:seeds/v2/screens/profile_screens/guardians/interactor/guardians_bloc.dart';
+
+class MyGuardiansTab extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return StreamBuilder<List<GuardianModel>>(
+        stream: BlocProvider.of<GuardiansBloc>(context).guardians,
```

Comment:
Now I understand why this guardians is a getter and is not initialized in the bloc constructor as I do before.